system-addon/content-src/components/Card/_Card.scss

```diff
@@ -255,10 +257,11 @@
       $container-size: 32px;
       background-color: var(--newtab-card-background-color);
       border-radius: $container-size / 2;
+      clip-path: inset(0 0 $container-size - ($card-height-compact - $card-preview-image-height-compact - 2 * $card-detail-vertical-spacing));
```

Comment:
Hmmm, I understand this produces the necessary `4px`, but I think this should be `$container-size - $icon-size - $card-detail-vertical-spacing`, that is, height of `.card-context` minus height of overlap with `.card-preview-image` minus the top padding of `.card-details`?